NEWS

```diff
@@ -2,6 +2,27 @@
 
 Sandia OpenSHMEM NEWS -- history of user-visible changes.
 
+v1.5.3rc1
+---------
+- Added several enhancements to better support SOS as a backend for Intel® SHMEM.
+- Added extension support for GPU RDMA and external heap creation.
+- Added support for multi-NIC configurations via libfabric. The feature is
+  enabled by default. It can be disabled with the environment variable,
+  SHMEM_OFI_DISABLE_MULTIRAIL=1.
+- Added initial support for multi-NIC topology optimizations via hwloc.
+  Detection of hwloc is enabled by default. It can be disabled with the
+  configuration flag, --without-hwloc.
```

Comment:
```suggestion
  configuration flag, --with-hwloc=no.
```

Reply:
Nevermind. I think both work (not getting error of unrecognized flag when using **--without-hwloc**).